crescita-personale/tirocinio.md

```diff
@@ -0,0 +1,148 @@
+crescita personale: capirete insieme dove sei oggi e dove vuoi essere al termine del tirocinio, e 
+come misurare se stai facendo progressi in questa direzione. Con il tuo mentore avrete inoltre dei 
+meeting 1:1 (uno-a-uno) ogni due venerdì per valutare come sta andando.
+
+## Lavorare per il cliente
+
+Ora che ti sei sistemato e hai tutti gli strumenti che ti servono, è ora di iniziare il lavoro! 
+Verrai inserito a tutti gli effetti nel team del progetto, dunque parteciperai a eventuali meeting
+col cliente e con il team. Il tuo mentore e il team del progetto ti assegneranno alcuni task da 
+completare. 
+
+Sarai seguito in ogni passo del processo, ma se hai bisogno di aiuto con qualcosa non esitare a 
+chiedere a chiunque in messaggio privato o, ancora meglio, in #sviluppo.
+
+Il codice che scrivi sarà controllato dal tuo mentore e da un membro del team del progetto su cui 
+stai lavorando. Loro ti daranno consigli su come migliorare quello che hai fatto e ti suggeriranno
+risorse per lo studio in base agli aspetti da migliorare. 
+
+Sappiamo che, soprattutto per chi non è mai stato in un contesto lavorativo reale, trovarsi a
+lavorare per dei clienti può essere spaventoso, se non terrificante! Ecco qualche pillola per
+rassicurarti:
+
+- Con tutti i nostri clienti abbiamo un ottimo rapporto, spesso costruito in anni di lavoro insieme.
+Se lavoriamo ancora con loro, è perché sono persone fantastiche che non ti mangeranno.
+- Se ti abbiamo scelto per lavorare con noi, è perché pensiamo che tu possa farcela.
+- Il tuo mentore e il team del progetto a cui sei stato assegnato ti seguiranno e ti supporteranno
+in qualunque dubbio o problema tu dovessi avere. Non aver paura di chiedere aiuto!
+- Sei qui per imparare, dunque ci aspettiamo (anzi, speriamo) che tu faccia degli errori: è il solo
+modo in cui si impara veramente.
+- Abbiamo creato dei processi (es. le code review) per evitare che eventuali tuoi errori possano 
+danneggiare il cliente, ma se anche dovessi rompere qualcosa, non sarà la fine del mondo.
+
+Happy coding!
+
+## Dopo il tirocinio
+
+Al termine dei quattro mesi di tirocinio, avrai un incontro col tuo mentore per valutare il 
+progresso che è stato fatto. Se ci siamo trovati bene e pensiamo che Nebulab sia il posto per te, ti 
+offriremo di entrare a pieno titolo nel team come sviluppatore junior. 
+
+Ecco cosa accadrà se dovessi accettare:
+
+- Sarai spostato su un altro progetto, più adatto al tuo nuovo livello di competenza
+- Ti inviteremo a visitare l'ufficio più vicino a te per incontrare il team (se hai fatto il 
+tirocinio da remoto)
+- Ti porteremo fuori a cena per festeggiare
+
+Una volta sopravvissuto/a alla cena, il resto della transizione dovrebbe essere piuttosto facile!
+
+## Domande frequenti
+
+**Il tirocinio è pagato?**
+
+**Sono un programmatore esperto, ma in un altro ambiente. Posso comunque fare il tirocinio?**
```

Comment:
Per me via libera, magari possiamo prevedere ed esplicitare che ci sono dei casi particolari in cui magari la formula del tirocinio va un po' stretta. tipo: `Se non rientri nelle categorie di partecipanti ideali ma pensi di essere comunque un buon fit per il tirocinio contattaci e discutiamone insieme`.